clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10153,11 +10161,18 @@ void OffloadBundler::ConstructJobMultipleOutputs(
       StringRef Val(
           TCArgs.getLastArg(options::OPT_fsycl_force_target_EQ)->getValue());
       llvm::Triple TT(C.getDriver().getSYCLDeviceTriple(Val));
-      Triples += TT.normalize(
-        llvm::Triple::CanonicalForm::FOUR_IDENT);
-    } else
-      Triples += Dep.DependentToolChain->getTriple().normalize(
-        llvm::Triple::CanonicalForm::FOUR_IDENT);
+      Triples += TT.normalize();
```

Comment:
Why does this case not warrant a 'HIP only' variant?

Thanks

Reply:
It's because we are already in a SYCL specific offloading mode, so HIP never hits here.

Reply:
ah....thanks